### PR TITLE
Add Jaeger Tracing chapter, minor fixes

### PR DESCRIPTION
**Pull request by calohmn**
Also remove the "experimental" status for the CoAP adapter (fully supported since Hono 1.5).

### CODE REVIEW

**Comment by calohmn** [comment]
Also fix repo URLs in the ".github" directory, replacing the obsolete "https://kubernetes-charts.storage.googleapis.com" location and adding prometheus and grafana repos.

**Comment by sophokles73** [comment]
FMPOV we should split this up into two PRs one for updating/fixing the URLs in the repo itself and another one dealing with the changes/fixes in the Hono chart.

**Review by sophokles73 — COMMENTED** [review]
FMPOV we should split this up into two PRs one for updating/fixing the URLs in the repo itself and another one dealing with the changes/fixes in the Hono chart.

**Comment by calohmn** [comment]
Yes, I'll create a separate PR.

File: charts/hono/README.md [hunk 1]
```diff
@@ -392,18 +392,49 @@ The following table provides an overview of the corresponding configuration prop
...
+to `true` when running Helm:
+
+~~~sh
+helm install --dependency-update -n hono --set jaegerBackendExample.enabled=true eclipse-hono eclipse-iot/hono
+~~~
+
+This will create a Jaeger back end instance suitable for testing purposes and will configure all deployed Hono components to use the
+Jaeger back end.
+
+Note that this can only be used with the standard Hono images published on Docker Hub with version 1.5.0 or greater. Custom
```
Comment (sophokles73) [inline charts/hono/README.md:423]: 1.5.0 or later.

File: charts/hono/README.md [hunk 2]
```diff
@@ -392,18 +392,49 @@ The following table provides an overview of the corresponding configuration prop
...
+images must have been built with the Jaeger client included. For the standard Hono components that means using the
+`jaeger` Maven profile for the build.
+
+The following command can then be used to return the IP address with which the Jaeger UI can be accessed in a
```
Comment (sophokles73) [inline charts/hono/README.md:427]: the IP address at which the Jaeger UI can be ...

**Comment by calohmn** [comment]
PR to fix the CI checks is #182.

**Comment by calohmn** [comment]
Rebased and applied suggested changes.

**Pull request merged**